### ISSUE TITLE
Failed trying to dockerize pyrite

### ISSUE THREAD

**Issue by NopeNix** [garage44/pyrite#93]
Hi

i failed to dockerize pyrite because it uses docker on it's own. This makes it too complicated for me to generate my own docker stack or rewrite your code to be run fully in containers. I played around with the thought of docker in docker but after some research it turned out that docker-in-docker is not a good advice in general so i stopped my try for now.

Can you create a container and provide a docker-compose example?

i don't know if this helps but i already tried to create a dockerfile, of course not working if fails on line 18 makes sense because there is no docker inside that container but maybe useful for the devs if anyone wants to catch up on this request :-) 
```dockerfile
FROM alpine

RUN apk update
RUN apk add --no-cache git
RUN apk add --no-cache npm
RUN apk add --no-cache py3-pip
RUN apk add --no-cache docker-cli python3 
RUN apk add --no-cache py-pip python3-dev libffi-dev openssl-dev gcc libc-dev rust cargo make
RUN apk add --no-cache curl

RUN mkdir /app
WORKDIR /app

RUN pip3 install docker-compose
RUN git clone https://github.com/garage44/pyrite.git
WORKDIR /app/pyrite
RUN npm i
RUN npm run galene
RUN nodemon admin/app.js

CMD npm run dev
```

**Comment by jvanveen** [comment]
Hey,

Thanks for the docker example. I'll see what I can do.

**Comment by NopeNix** [comment]
@jvanveen thanks a lot for helping :-) this looks like as it would full fill my needs. I'll be able to test at the weekend and let you know the results, thanks again!

**Comment by jvanveen** [comment]
The initial group creation is kinda fixed, but I bumped Galene to 0.5, which probably needs some additional changes. Conferences are broken now. Will look into that asap.

**Comment by jvanveen** [comment]
@NopeNix I think the Docker flow is finished now. There is a simpler command to test/build the Docker image now:
```bash
npm run build-docker
```
Instead of building the Docker image from npm using npx, it now just packs the current frontend files and adds it to the Pyrite docker image. This makes it much easier to test the build with Docker. Still need to fix the 0.5 Galene build though. I'll keep this ticket open until that's fixed.

**Comment by jvanveen** [comment]
Protocol file from Galene was updated; permissions are now coming in as an Array instead of an Object; that's why the video streams were broken. Should work ok again.
